src/common/storage/backends/udcp/Client.js

```diff
@@ -0,0 +1,80 @@
+/* globals define */
+import {Storage} from "../Storage"
```

Comment:
Change this to use requirejs for importing. (This one is likely going to throw an error as I doubt `import` statements are supported in this version.)